cli/tests/cast.rs

```diff
@@ -0,0 +1,27 @@
+//! Contains various tests for checking cast commands
+use std::env;
+
+use foundry_cli_test_utils::{
+    casttest,
+    util::{TestCommand, TestProject},
+};
+
+// tests that the `cast find-block` command works correctly
+casttest!(finds_block, |_: TestProject, mut cmd: TestCommand| {
+    // Construct args
+    let timestamp = "1647843609".to_string();
+    let eth_rpc_url = env::var("ETH_RPC_URL").unwrap();
```

Comment:
You should check if this URL is set or not, since when running tests locally that might not be the case - if it is not set emit a warning or something, see https://github.com/gakonst/foundry/blob/master/cli/test-utils/src/macros.rs#L119-L123

Reply:
> Ah got it - was assuming the github actions exposed an `ETH_RPC_URL` env variable

It does, but this is not necessarily the case locally :smile:

Reply:
Fixed :)